src/styles/blocks/page.scss

```diff
@@ -0,0 +1,25 @@
+.page {
+  font-family: "Space Mono", sans-serif;
```

Comment:
Consider using a variable for the font-family to make it easier to update in the future.